cmd/kubeadm/test/token_test.go

```diff
@@ -28,12 +28,11 @@ const (
 	TokenExpectedRegex = "^\\S{6}\\:\\S{16}\n$"
 )
 
-var kubeadmPath string
+var kubeadmPath = flag.String("kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
```

Comment:
correct. as a rule, package-level unit tests should be self-contained, not depend on flags or external built binaries. 

exceptions live in our e2e and integration packages, not sprinkled through the cmd or pkg packages.

these two tests really seem like they should follow the test-cmd.sh pattern, building a binary and exercising invoking and testing the output via a shell script, rather than live as package unit tests